test/metabase/cmd_test.clj

```diff
@@ -13,51 +13,29 @@
     (is (fn? the-fxn))))
 
 (deftest import-test
-  (with-redefs [cmd/resolve-enterprise-command (constantly
-                                                (fn [& args]
-                                                  (cons 'f args)))]
+  (with-redefs [cmd/call-enterprise list]
     (testing "load (v1)"
       (testing "with no options"
-        (is (= '(f "/path/" {:mode :skip, :on-error :continue})
+        (is (= '(metabase-enterprise.serialization.cmd/v1-load "/path/" {:mode :skip, :on-error :continue})
                (cmd/load "/path/"))))
       (testing "with options"
-        (is (= '(f "/path/" {:num-cans :2})
-               (cmd/load "/path/" "--num-cans" "2")))
-        (testing "People can still set --v2 true"
-          (is (= '(f "/path/" {:v2 :true})
-                 (cmd/load "/path/" "--v2" "true"))))))
+        (is (= '(metabase-enterprise.serialization.cmd/v1-load "/path/" {:mode :skip, :on-error :continue :num-cans :2})
+               (cmd/load "/path/" "--num-cans" "2")))))
     (testing "import (v2)"
       (testing "with no options"
-        (is (= '(f "/path/" {:mode :skip, :on-error :continue, :v2 :true})
```

Comment:
v2 doesn't really have any real documented, working options at the moment, so we can add these back to the tests as the feature solidifies